### DISCUSSION TITLE
The Foggiest Clue

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#54] [Feedback]
### New Textures
![Funny dropper screenshot](https://user-images.githubusercontent.com/1903667/147719653-4f02a714-0135-40cb-b15b-0e66030c7dec.png)
- Dropper front horizontal
- Brewing stand
- Chorus flower (dummy/dev placeholder)
- Chorus plant
- Double tall sea grass (unanimated placeholder)
- Red coral plant
- Sea pickle
- Conduit (but its PBR textures are not supported in Minecraft?)

### Updated Textures
![Updated textures screenshot](https://user-images.githubusercontent.com/1903667/147719245-0eb3758b-0686-48dd-80b0-4b21a213bff8.png)

- Raw ores
- Still water animation
- Stone andesite and diorite
- Blue and pink coral blocks
- Blue, lime, and orange glazed terracotta
- Purpur blocks and pillar
- End stone and End bricks
- Simple, clear glass

### Important Fog Fix
![Distance fog example](https://user-images.githubusercontent.com/1903667/147717876-9e4c9a52-b67e-4ad4-9e84-6d68bec9f814.png)
![Water fog example](https://user-images.githubusercontent.com/1903667/147717854-dad0dbad-1fed-4cc0-ac5b-6a640d930228.png)

> **"Oooh, so _that's_ how volumetric fogs work..."**
> I completely misunderstood volumetric fog (and it was apparent). The `max_density_height` property measures the fog height in blocks, *from the ground up*. I mistakenly thought this value was supposed to be set to the desired Y coordinate. 🤦‍♂️

- All biomes have fog settings
- Fogs are generally better, but not all biomes have been fully tested or tweaked.

<hr /><em>This discussion was created from the release <a href='https://github.com/jasonjgardner/jg-rtx/releases/tag/0.22.0'>The Foggiest Clue</a>.</em>

**Comment by creepcomix** [comment]
Wow! Great job! Looks SUPER good!

**Reply by jasonjgardner** [reply]
Thanks so much!

**Comment by justincrapse** [comment]
Excited to see the new blocks, updates, and fog.
Beautiful work

**Reply by jasonjgardner** [reply]
Thank you! Thoughts on the new glass? I agree that only _some_ of the previous glass textures worked well in practice. I think this new glass will work in a bigger variety of builds, but I'm reluctant to lose too much style and detail.

**Reply by justincrapse** [reply]
Hi Jason, I'll have to check it out soon. I've been in the trenches with my server and my Texture pack development, but taking a break. It may make more sense to have simpler glass blocks for vanilla then offer an addon for new glass blocks. But... those blocks are so good looking, so your call. That is my struggle with some blocks... they look too cool, but aren't very vanilla.
